src/pages/userPage/ActiveMembership/ActiveMembership.tsx

```diff
@@ -82,7 +82,10 @@ const ActiveMembership = () => {
       notificationLogic("error", error.message);
     });
 
-    setAccessLevels(await activeMembershipApi.getAccessLevelById(userId));
+    await activeMembershipApi.getAccessLevelById(userId).then(async (response) => {
```

Comment:
You don't need the brackets around the 'response' variable

Reply:
In whole file before my changes is brackets. So maybe it`s better to leave like that?)